.github/workflows/release-notes-sync.yaml

```diff
@@ -0,0 +1,21 @@
+name: Sync release notes with nebari.dev/docs
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@master
+      - name: Run Release File Sync ♻️
+        uses: BetaHuhn/repo-file-sync-action@v1
+        with:
+          GH_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does anyone know if `secrets.GITHUB_TOKEN` has permission to `nebari-dev/nebari-docs`?
If not, do we need a new secret for this?

Reply:
@iameskild Looks like it doesn't have access, ref: https://github.com/nebari-dev/nebari/actions/runs/3597037385/jobs/6058376405#step:3:15

I see we use [`secrets.GH_PAT` in nebari-dev/.github](https://github.com/nebari-dev/.github/blob/main/.github/workflows/sync-issue-templates.yaml#L23) -- @trallard is this `secrets.GH_PAT` an org-level secret that we can use here? If not, we can create a new secret with the required scope.